### PR TITLE
reorder feedback and support page

### PR DESCRIPTION
**Pull request by nloding**
## Description

Move the community forum link below the official support channels on the Feedback & Support page. Changed the wording slightly to be a bit more explicit that the forum is not an official support channel but rather a community forum.

Background: A user opened a [bug report](https://github.com/camunda/zeebe/issues/13832) in the Zeebe repo, and because they were using SaaS we referred them to "our dedicated support team" ... the link to "our dedicated support team" goes to the Feedback & Support page, and the user then [started a thread](https://forum.camunda.io/t/oauth-token-refresh-can-trigger-rate-limiting-and-log-errors/47260) on the forum. However, some wires were crossed because if I saw the forum post I would have asked them to file a bug on GitHub!

The DevRel team thought that sliding the forum down on the docs page may make the support options clearer. While it may not have made a huge difference in this users support journey, maybe it will help in the future!

(Let me know if I need to version this change and/or backport it.)

## When should this change go live?

- [ ] This change is not yet live and should not be merged until {ADD_DATE} (apply `hold` label or convert to draft PR)?
- [x] There is no urgency with this change.
- [ ] This change or page is part of a marketing blog, conference talk, or something else on a schedule.
- [ ] This functionality is already available but undocumented.
- [ ] This is a bug fix or security concern.

## PR Checklist

- [ ] I have added changes to the relevant `/versioned_docs` directory, or they are not for an **already released version**.
- [ ] I have added changes to the main `/docs` directory (aka `/next/`), or they are not for **future versions**.
- [ ] My changes require an [Engineering review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [ ] My changes require a [technical writer review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

**Comment by akeller** [comment]
@nloding, I made some changes in the `/docs` directory. You'll see I actually deleted the feedback page in the Console section. 

Can you look at how we present the forum on the Contact page? Does it still make sense to channel bug and feature requests through the forum or is there another mechanism or positioning you'd like to have?

**Comment by christinaausley** [comment]
How can I best assist here? Looks like @akeller made a majority of changes, but do we just need a review, or additional adjustments I can help with?

**Comment by akeller** [comment]
@christinaausley good question - I'd like to take this as an opportunity to route all links out to feedback/support to the contact page and make sure that's updated. DevRel will need to ultimately tell us how they want the forum presented. 

Notable exception(s) - Modeler has its own category monitored by the Modeler teams. Onboarding and Web Modeler have experiments or initiatives where they collect feedback through the forum. 

Can you find any others?

**Comment by christinaausley** [comment]
@akeller Do you mind if I launch a separate issue for this so I can double check with engineering teams and do a little more research here?

**Comment by akeller** [comment]
@nloding checking in on this PR before we get into the pre-minor crunch. If the Contact page positions the forum how DevRel would like moving forward, we can handle the rest of the changes on your behalf.

**Comment by christinaausley** [comment]
@nloding Let me know if I can assist, even just with the merge conflicts 👍

**Comment by nloding** [comment]
@christinaausley & @akeller - naming fixed, merge conflicts fixed, should be good to go!

File: src/pages/contact.md [hunk 1]
```diff
@@ -7,12 +7,12 @@ description: Contact Camunda, submit feedback, or find support using the Camunda
 
 There are various channels where you can reach us.
 
-- We encourage everyone to participate in our **community** via the [Camunda Platform community forum](https://forum.camunda.io/), where you can exchange ideas with other Zeebe and Camunda Platform 8 users, as well as the product developers.
+- We encourage everyone to participate in our **community** via the [Camunda community forum](https://forum.camunda.io/), where you can exchange ideas with other Camunda users, as well as the product developers.
 
 - We welcome your **bug** reports and **feature requests** through our community channels mentioned above.
```
Comment (akeller) [inline src/pages/contact.md:12]: @nloding is DevRel ok with this line?

File: versioned_docs/version-8.2/components/console/console-troubleshooting/feedback-and-support.md [hunk 2]
```diff
@@ -3,8 +3,8 @@ id: feedback-and-support
 title: Feedback and support
```
Comment (akeller) [inline versioned_docs/version-8.2/components/console/console-troubleshooting/feedback-and-support.md:3]: This page should be removed entirely. let's route all contact (problems, questions, suggestions, etc.) through the Contact page to make it easier to maintain.

File: src/pages/contact.md (hunk 1 above)
Comment (nloding) [reply src/pages/contact.md:12]: Made a slight change after getting some more feedback, should be solid now!

**Comment by akeller** [comment]
I'll fix the build issues and any remaining cleanup 😄

**Review by akeller — APPROVED** [review]
10/10 please collaboration with us again 😃

**Pull request merged**